### PR TITLE
Add Euruko videos

### PR DESCRIPTION
**Pull request by marcoroth**
CleanShot 2024-08-07 at 14 09 41

### CODE REVIEW

File: data/euruko/euruko-2023/videos.yml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+- title: 'Winner Ceremony'
+  raw_title: Euruko 2023. Winner Ceremony
+  speakers:
+    - Sergey Sergyenko
+    - Muhamed Isabegović
+  event_name: Euruko 2023
+  published_at: '2023-10-04'
+  description: ''
+  video_id: IlaLqll1rZA
```
Comment (adrienpoly) [inline data/euruko/euruko-2023/videos.yml:299]:
```suggestion
```

I suggest we remove this one

File: data/euruko/euruko-2017/videos.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  description: 'Slides: https://euruko2017.org/downloads/slides/search_engine.pdf'
+  video_id: 4zanERox264
+
+- title: 'City Pitches'
+  raw_title: Hosting City Pitches @EuRuKo2017
+  speakers: []
+  event_name: Euruko 2017
+  published_at: '2017-12-22'
+  description: ''
+  video_id: "-z9s743MUiw"
```
Comment (adrienpoly) [inline data/euruko/euruko-2017/videos.yml:85]:
```suggestion
```

I suggest we remove this one

File: data/euruko/euruko-2022/videos.yml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+- title: Lightning Talks & City Pitches
+  raw_title: Lightning talks & city pitches
+  speakers:
+    - Renato dos Santos Cerqueira
+    - Mohnish Jadwani
+    - Hans Schnedlitz
+    - Hana Harencarova
+    - Antti Merilehto
+    - Tomasz
```
Comment (adrienpoly) [inline data/euruko/euruko-2022/videos.yml:106]:
```suggestion
    - Tomasz Cudziło
```
I think it is him This being said shouldn't we list only the speakers of the lighting talks?

- Renato dos Santos Cerqueira    
- Mohnish Jadwani
- Hans Schnedlitz
- Hana Harencarova

File: data/euruko/euruko-2023/videos.yml [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+    - Paweł Strzałkowski
+    - Johnny Shields
+    - Alekander
+    - Mariusz Kozieł
+    - Adrian Marin
+    - Yaroslav Shmarov
+  event_name: Euruko 2023
+  published_at: '2023-10-04'
+  description: ''
+  video_id: J7oargktOEw
```
Comment (adrienpoly) [inline data/euruko/euruko-2023/videos.yml:31]:
```suggestion
```

those are only the pitches I suggest we remove them

File: data/euruko/euruko-2023/videos.yml [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+    - Youssef Boulkaid
+    - Josua Schmid
+    - Masafumi Okura
+    - Miron Marczuk
+    - Chris Hasiński
+    - Yakau
+  event_name: Euruko 2023
+  published_at: '2023-10-04'
+  description: ''
+  video_id: t6_uvjX25Jk
```
Comment (adrienpoly) [inline data/euruko/euruko-2023/videos.yml:122]:
```suggestion
```

those are only the pitches I suggest we remove them

File: data/euruko/euruko-2023/videos.yml [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+  video_id: syRtJsFdTvI
+
+- title: 'City Pitch: Verona'
+  raw_title: City Pitching. Verona
+  speakers:
+    - Giorgio Fochesato
+  event_name: Euruko 2023
+  published_at: '2023-10-04'
+  description: ''
+  video_id: usrc27rytMk
```
Comment (adrienpoly) [inline data/euruko/euruko-2023/videos.yml:256]:
```suggestion
```

Those are only city pitches I suggest we remove them

**Review by adrienpoly — COMMENTED** [review]
a few suggestions of videos to remove

File: data/euruko/euruko-2022/videos.yml (hunk 3 above)
Comment (marcoroth) [reply data/euruko/euruko-2022/videos.yml:106]: Yeah, not sure. Happy either way

**Comment by marcoroth** [comment]
Good points, I removed the non-talk videos 👍🏼

File: data/euruko/euruko-2022/videos.yml [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+    - Ufuk Kayserilioglu
+    - Sergey Sergyenko
+    - Andy Croll
+    - Vesa Vänskä
```
Comment (adrienpoly) [inline data/euruko/euruko-2022/videos.yml]:
```suggestion
```

Comment (adrienpoly) [inline data/euruko/euruko-2022/videos.yml]:
```suggestion
```

**Comment by adrienpoly** [comment]
live

**Pull request merged**